client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -4,13 +4,20 @@ import CircularProgress from '@material-ui/core/CircularProgress';
 import useStyles from './useStyles';
 import { useAuth } from '../../context/useAuthContext';
 import { useSocket } from '../../context/useSocketContext';
-import { useHistory } from 'react-router-dom';
+import { useHistory, Switch, Route } from 'react-router-dom';
 import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
 import { useEffect } from 'react';
+import { Box, Typography } from '@material-ui/core';
```

Comment:
These two imports are never used

Reply:
I have fixed this in my next pull request.